flowetl/flowetl/flowetl/operators/staging/append_sightings_to_main_table.py

```diff
@@ -0,0 +1,20 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from airflow.operators.postgres_operator import PostgresOperator
+
+
+class AppendSightingsToMainTable(PostgresOperator):
+    """
+    Attaches reduced.sightings_DATE to the main reduced.sightings table as a partition, dropping the date
+    constraint as it does so.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            task_id="AppendSightingsToMainTable",
```

Comment:
I think setting the task ID should really be left to the DAG definition, rather than hard-coded in the operator class (i.e. set the `task_id` argument in `create_staging_dag`, not here).

Reply:
In fact, I'm not entirely convinced it's worth defining these custom operator classes at all, if they're just wrapping a sql script - could use `PostgresOperator`s directly in `create_staging_dag` (i.e. `append_sightings_to_main_table = PostgresOperator(task_id="AppendSightingsToMainTable", sql="append_sightings_to_main_table.sql")`).

Reply:
I was wondering it while I was making them - the thinking was that any preprocessing of parameters before applying the template can live in the custom operators, but those turned out not to be needed in pretty much every case, or Jinja could take care of it. I'll take them out.